portfolio/src/main/java/com/google/sps/Data/User.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** A user and its log in credentials, used to authenticate that user. */
+public final class User {
+
+  private final boolean loggedIn;
+  private final String logUrl; // URL used for either login or logout.
```

Comment:
I would say this URL shouldn't be modeled as part of an object named 'User', as it's not really intrinsic to the user. Perhaps it could be a part of an object named 'UserAuthenticationData', or something similar. You don't have to make a change to this here or now, I'm just remarking on object-oriented design.

Reply:
Oh, nice to know! I will make the change :)

Reply:
Done!